src/frontend/src/states/UserState.tsx

```diff
@@ -167,5 +168,8 @@ export const useUserState = create<UserStateProps>((set, get) => ({
   },
   hasViewRole: (role: UserRoles) => {
     return get().checkUserRole(role, UserPermissions.view);
+  },
+  canApproveOrder: (order: number) => {
+    return false;
```

Comment:
Is this a placeholder?

Reply:
This looks like leftovers from an experiment they slipped through the cracks. I'll remove it if I don't re-discover why I added it.